### ISSUE TITLE
key_press/krey_release events

### ISSUE THREAD

**Issue by dawikur** [cnjinhao/nana#129]
Hi,

I am using `form.events().key_press[release]()` to subscribe to keyboard events. Unfortunately when I press any key and hold it down I get lots and lots of `key_press/key_release` calls - simply as OS is repeating key. This is not behavior I want and would expect - I need one press and one release.

Is this known issue? Or maybe this is `by design`? Can I change the behavior?

I am running linux
```bash
> uname -a
Linux Archangel 4.5.0-1-ARCH #1 SMP PREEMPT Tue Mar 15 09:41:03 CET 2016 x86_64 GNU/Linux
```

**Comment by cnjinhao** [comment]
Thank you for report this issue. The behavior of key_press is correct, key_release's is wrong on Linux. I will fix this issue. And then you can make one press with a flag.
`
bool excl_repeat_key_press = false;
widget.events().key_press([&excl_repeat_key_press]
{
    if(!excl_repeat_key_press)
    {
          //One press;
          excl_repeat_key_press = true;
    }
});

widget.events().key_release([&excl_repeat_key_press]
{
    excl_repeat_key_press = false;
});
`

**Comment by cnjinhao** [comment]
This issue is fixed, see 9f61e21c87243ed0a209b9a8fe487b6a3bdd9700

**Comment by dawikur** [comment]
Great, tested with fix. Works great, thanks!
